ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -207,13 +207,13 @@ instance TPraosCrypto sc => TranslateNetworkProtocolVersion (CardanoBlock sc) wh
 
   nodeToNodeProtocolVersion _ = \case
       CardanoNodeToNodeVersion1 -> NodeToNodeV_1
-      CardanoNodeToNodeVersion2 -> error "NodeToNodeV_2"
+      CardanoNodeToNodeVersion2 -> NodeToNodeV_1
```

Comment:
Is this right? Shouldn't it be set to `NodeToNodeV_2` which is added in this PR.

Reply:
Of course :facepalm: